lis/surfacemodels/land/noahmp.4.0.1/da_snow/noahmp401_snow_update.F90

```diff
@@ -150,7 +151,8 @@ subroutine noahmp401_snow_update(n, t, dsneqv, dsnowh)
   ! alter snow depth change to be in direction of SWE change
```

Comment:
If we update the default snow update routine, we should 
(1) change the comments so that it is clear that SWE changes in the direction of the snow depth change
(2) highlight that the name "snow_update" is used here for "snow_depth_update" (i.e. we use this routine to assimilate SD and let SWE depend on it, rather than the other way around).

Reply:
Thx! Made corresponding changes.